src/vs/base/browser/ui/iconLabel/iconLabel.ts

```diff
@@ -24,8 +24,13 @@ export interface IIconLabelCreationOptions {
 	hoverDelegate?: IHoverDelegate;
 }
 
+export interface IIconLabelMarkdownString {
+	markdown: IMarkdownString | Promise<IMarkdownString | string | undefined> | string;
+	fallback: string | undefined;
```

Comment:
From the name it is not immediately obvious to me when `fallback` is used. Either needs good documentation or another name?

Reply:
Renamed.